plonky2/src/iop/generator.rs

```diff
@@ -328,35 +328,27 @@ impl<F: Field> SimpleGenerator<F> for NonzeroTestGenerator {
 
 /// Generator used to fill an extra constant.
 #[derive(Debug, Clone)]
-pub struct ConstantGenerator<F: Field> {
+pub(crate) struct ConstantGenerator<F: Field> {
     pub gate_index: usize,
     pub constant_index: usize,
-    pub target_index: usize,
+    pub input_index: usize,
```

Comment:
nit: Hm why the "input" name here? I would actually see this index as more of an output index, which would be consistent with the naming `wire_output()` used in `ConstantGate`.

Reply:
Hm I see what you mean, I just thought I should match the `Wire::input` naming - in that sense each wire is an input to some gate. But I see how it can be misleading.

Do you think "wire" would be more clear? Or maybe "column" - it's not often used with circuits but at least it couldn't be confused with anything else.

Reply:
Ah I see, that makes sense. Yeah I think "wire" might be clearer.